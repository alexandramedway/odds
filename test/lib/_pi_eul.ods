do print(PI)
do print(EUL)
