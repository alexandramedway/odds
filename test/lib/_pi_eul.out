3.14159265359
2.71828182846
